packages/charting/src/components/DonutChart/DonutChart.base.tsx

```diff
@@ -151,7 +149,7 @@ export class DonutChartBase extends React.Component<
         };
         return legend;
       });
-    const legends = <Legends legends={legendDataItems} />;
+    const legends = <Legends legends={legendDataItems} centerLegends={true} />;
```

Comment:
Nit: this can be `centerLegends` instead of `centerLegends={true}`.

Reply:
Cool. I didn't know this.